.circleci/config.yml

```diff
@@ -1,56 +1,91 @@
+references:
+  #
+  # Environment
+  #
+  shared-environment: &shared-environment
+    CIRCLE_ARTIFACTS: /tmp/artifacts
+    CIRCLE_TEST_REPORTS: /tmp/test_results
+
+  #
+  # Image
+  #
+  docker-image: &docker-image
+    - image: circleci/node:10.10.0
+
+  #
+  # Cache management
+  #
+  restore-git-cache: &restore-git-cache
+    name: Restore git cache
+    keys:
+      - v2-git-{{ .Branch }}-{{ .Revision }}
+      - v2-git-{{ .Branch }}
+      - v2-git
+  save-git-cache: &save-git-cache
+    name: Save git cache
+    key: v2-git-{{ .Branch }}-{{ .Revision }}
+    paths:
+      - ".git"
+
+  restore-npm-cache: &restore-npm-cache
+    name: "Restore npm cache"
+    keys:
+       - v1-npmcache-{{ checksum ".nvmrc" }}-{{ checksum "npm-shrinkwrap.json" }}
+       - v1-npmcache-{{ checksum ".nvmrc" }}
+  save-npm-cache: &save-npm-cache
+    name: "Save npm cache"
+    key: v1-npmcache-{{ checksum ".nvmrc" }}-{{ checksum "npm-shrinkwrap.json" }}
+    paths:
+      - ~/.npm
+
+
 version: 2
 jobs:
-  build:
-    parallelism: 6
+  #
+  # Shared setup required for all jobs
+  #
+  setup:
+    docker: *docker-image
     working_directory: ~/wp-calypso
-    docker:
-      - image: circleci/node:10.10.0-browsers
-        environment:
-          CIRCLE_ARTIFACTS: /tmp/artifacts
-          CIRCLE_TEST_REPORTS: /tmp/test_results
-
     steps:
-      - restore_cache:
-          name: Restore git cache
-          key: v1-20180524-git
-
+      - restore_cache: *restore-git-cache
       - checkout
-
-    # Update origin/master so we can diff against it later
+      # Update origin/master so we can diff against it later
       - run:
           name: Update master branch
           command: git fetch --force origin master
+      - save_cache: *save-git-cache
 
-      - save_cache:
-          name: Save git cache
-          key: v1-20180524-git
-          paths:
-            - ".git"
+      - restore_cache: *restore-npm-cache
+      - run: npm ci
+      - save_cache: *save-npm-cache
+      - persist_to_workspace:
+          root: .
+          paths: .
+
+  #
+  # Build translations
+  #
+  translate:
+    parallelism: 6
```

Comment:
i think this can just be `1`. translate isn't parallelized.

Reply:
Yes!

Reply:
0d978540fc84c1ad99b49d515e32df97b8ec913c